### PR TITLE
Register page region.

### PR DESCRIPTION
**Pull request by afrisalyp**
https://github.com/topcoder-platform/community-app/issues/2866

### CODE REVIEW

**Comment by afrisalyp** [comment]
Correlated work with
https://github.com/topcoder-platform/community-app/issues/2897.
Commit added.

**Comment by mishacucicea** [comment]
@afrisalyp Please don't commit space/tab changes, it's hard to follow the actual code change, also this leads to more conflicts. Remove the introduced spaces and resolve the conflict.

**Comment by afrisalyp** [comment]
@mishacucicea 
Thanks for the review.
Resolving the conflict.

I'll remove the space as much as possible but for these changes, we need to insert a div as a parent of the form region. https://github.com/appirio-tech/accounts-app/pull/257/files#diff-7b4625c50ae32579225e8ffd2be3b00eR7

Therefore, we need to adjust all the elements inside the new div.
We have to do it since `jade` uses whitespace and indentation as a part of the syntax.

**Comment by afrisalyp** [comment]
@mishacucicea done.
